lib/moonshot/controller.rb

```diff
@@ -82,8 +85,13 @@ def doctor
 
     def ssh
       run_plugins(:pre_ssh)
-      stack.ssh
-      run_plugins(:post_ssh)
```

Comment:
I didn't want to tackle the issue of whether or not we actually want pre-ssh/post-ssh hooks in this patch so I left it functionally equivalent.